<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_StartQuiz">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Quiz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Question">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Category">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Answer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NextQuestion">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ShowAnswer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Answer</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuestionNumber">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Question Number</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
